pkg/router/general.go

```diff
@@ -0,0 +1,136 @@
+package router
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+// @Summary Get Shop Info
+// @Description Get shop information with shop ID
+// @Tags Shop
+// @Accept json
+// @Produce json
+// @Param id path int true "Shop ID"
+// @Success 200 
+// @Failure 401 
+// @Router /shop/{id} [get]
+func getShopInfo(c echo.Context) error {
+    return c.NoContent(http.StatusOK)
```

Comment:
The indents are using 4 spaces instead of a tab. And there're some spaces before end of line too.
Find out using go fmt. Should we worry about this?

Reply:
yes, please format before submitting code

Reply:
@x06lan